server/index.html

```diff
@@ -3,38 +3,146 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>WebSockets</title>
+
+  <link rel="stylesheet" href="../css/styles.min.css">
+
+  <title>Tic-tac-toe</title>
 </head>
-<body>
-  <h1>WebSockets Test</h1>
+<script>
+  const startGame = () => {
```

Comment:
thinking about it, it might be more traditional to call this html file `public/index.html` and extract this js body as `public/index.js`